_posts/2015-07-15-code.md

```diff
@@ -39,3 +69,5 @@ int main(int argc, char const \*argv[])
 }
 
 {% endhighlight %}
+
+Note that when using the liquid tag, the `copy code` button is not displayed.
```

Comment:
do you know why this is the case? in principle, is it possible to make `copy_code.js` work with both types of code blocks?

Reply:
I am investigating this now, but apparently this is a [known issue](https://github.com/jekyll/jekyll/issues/6448) in Jekyll.

Under the hood though, I found out that this is the difference (when using the c++ example):
- Using markdown code block
```html
<pre class="highlight"><code>
```
- Using jekyll highlight
```html
<pre><code class="language-c--" data-lang="c++">
```

Maybe just removing the filter with `highlight` class works? Will try it now.